model/span.go

```diff
@@ -53,6 +55,14 @@ func (s *Span) HasSpanKind(kind ext.SpanKindEnum) bool {
 	return false
 }
 
+// GetSpanKind returns value of `span.kind` tag, empty string if not present
+func (s *Span) GetSpanKind() string {
```

Comment:
This seems tangential to the commit message...

Reply:
commit msg updated